src/platform/forms-system/src/js/web-component-patterns/addressPattern.jsx

```diff
@@ -527,7 +527,7 @@ export function addressUI(options) {
   if (!omit('postalCode')) {
     uiSchema.postalCode = {
       'ui:required': requiredFunc('postalCode', true),
-      'ui:title': 'Postal code',
+      'ui:title': options?.labels?.postalCode ?? 'Postal code',
```

Comment:
Just curious, what other label do we have for this?

Reply:
The Figma mocks have this as `Zip code` instead.